frappe/core/doctype/data_import/test_importer.py

```diff
@@ -56,22 +56,6 @@ def test_data_import_preview(self):
 		self.assertEqual(len(preview.data), 4)
 		self.assertEqual(len(preview.columns), 16)
 
-	def test_data_import_without_mandatory_values(self):
```

Comment:
Don't remove this test. Just change the assertion. Test whether there are errors if you try to import without mandatory fields.